src/redux/books/books.js

```diff
@@ -0,0 +1,45 @@
+// Actions
+const ADD_BOOK = 'bookstore/books/ADD';
+const REMOVE_BOOK = 'bookstore/books/REMOVE';
```

Comment:
- [ ] @elarhadu great work on your reducer, however, the action type should be like this example
 `const ADD_BOOK = bookstore/books/ADD_BOOK`, this will ensure you follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)